src/components/Semester/Semester.vue

```diff
@@ -280,11 +280,13 @@ export default defineComponent({
             // the `optOut` field.
             // Below, we find all the requirements it can possibly match,
             // and only remove the requirementID since that's the one we should keep.
-            const optOut = getAllEligibleRelatedRequirementIds(
+            const optOut = getRelatedRequirementIdsForCourseOptOut(
               crseId,
+              requirementID,
               store.state.groupedRequirementFulfillmentReport,
-              store.state.toggleableRequirementChoices
-            ).filter(it => it !== requirementID);
+              store.state.toggleableRequirementChoices,
+              store.state.userRequirementsMap
+            );
```

Comment:
This refers to dragging a course card from the requirements bar to the user's semesters. This should still work properly (edge cases need to be tested but I'm not sure what they are).

Reply:
Confirmed courses can drag to semesters (and fulfill multiple reqs at once like CS 1110).